snuba/web/db_query.py

```diff
@@ -547,7 +423,37 @@ def _get_query_settings_from_config(
     return clickhouse_query_settings
 
 
-def _raw_query(
```

Comment:
the `_raw_query` docstring is no longer accurate, please update it

Reply:
Let me know if the docstrings are appropriate, I moved most of it to `db_query()`